README.md

```
@@ -514,6 +514,57 @@ for peak in peaks:
+
+# Suppress informational messages
+peakbagger --quiet peak search "Mount Rainier"
+peakbagger -q peak show 2296
+```
+
+### Logging
+
+By default, the CLI shows minimal output. Use logging flags to see what's happening under the hood:
+
+**Verbose Mode (`--verbose` / `-v`)**: Shows HTTP requests with timing
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)